src/screens/components/Testimonials.js

```diff
@@ -2,49 +2,84 @@ import React from 'react'
 import styled from 'styled-components'
 
 import { TestimonialCard, Title } from '@/style-guide'
+import { secondary1, accent2 } from '@/style-guide/colors'
 
 const StyledContainer = styled.div`
+  background-color: ${secondary1};
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+`
+
+const StyledTestimonials = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
+
   @media (min-width: 760px) {
     flex-direction: row;
     justity-content: space-between;
     align-content: center;
   }
 `
 
+const StyledFooter = styled.div`
+  height: 100px;
+  width: 100%;
+  background-color: ${accent2};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Hammersmith One";
+
+  p {
+    font-weight: bold;
+    color: ${secondary1};
+  }
+`
+
 const Testimonials = () => (
   <>
-    <Title>
-      WHAT OUR CLIENTS ARE SAYING
```

Comment:
Let's add a bit more spacing between the top of the background text box and the below testimonial text. (see attached for mockup:

![Screen Shot 2019-05-24 at 6 03 38 PM](https://user-images.githubusercontent.com/17219379/58358763-4a438a80-7e4e-11e9-8dcf-29374cc28ae8.png))

Reply:
ack fixed